duckduckgo/Android
# Fix dialog theme on API 21/22

Task/Issue URL: https://app.asana.com/0/414730916066338/1201204276099174/f

### Description
This fixes the `AlertDialog` theming issue on API 21/22.

### Steps to test this PR

_On an API 21 or 22 emulator/device:_
1. Go to Settings.
2. Tap "Theme".
3. Verify that the dialog displays the correct Light theme.
4. Change the theme to "Dark".
5. Verify that the dialog displays the correct Dark theme.

### UI changes

**Light theme:**

| Before  | After |
| ------ | ----- |
![current_api_21](https://user-images.githubusercontent.com/3471025/137507236-c5a0d6a2-2b54-4863-95f6-db98de611d66.png)|![proposed_api_21](https://user-images.githubusercontent.com/3471025/137507250-a0086397-6a6a-440c-84b7-9345e7da6f8f.png)

**Dark theme:**

| Before  | After |
| ------ | ----- |
![current_api_21_dark](https://user-images.githubusercontent.com/3471025/137507372-c0c17d12-a5c9-46e0-9adf-31eb5040e7cc.png)|![proposed_api_21_dark](https://user-images.githubusercontent.com/3471025/137507384-ee7739e8-aebd-4283-9b56-5af328e4080b.png)
